UIKitTemplate/UIKitTemplate/Controllers/ThanksViewController.swift

```diff
@@ -3,11 +3,94 @@
 
 import UIKit
 
-/// Контроллер успешного заказа
-class ThanksViewController: UIViewController {
-//    override func viewDidLoad() {
-//        super.viewDidLoad()
-//
-//        // Do any additional setup after loading the view.
-//    }
+/// Экран успешного заказа
+final class ThanksViewController: UIViewController {
+    // MARK: - Constants
+
+    enum Constant {
+        static let nameCafeLabelText = "СПАСИБО \nЗА ЗАКАЗ"
+        static let adressInfoLabelText = """
+        Разскажи о насъ другу, отправь ему
+        промокодъ
+        на безплатный напитокъ и получи
+        скидку 10% на слѣдующій заказъ.
+        """
+        static let returnBackButtonText = "Хорошо"
+    }
+
+    // MARK: - Public Properties
+
+    weak var delegate: MuveToFirstDelegate?
+
+    // MARK: - Visual Components
+
+    private lazy var cancellationButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(systemName: "xmark"), for: .normal)
+        button.frame = CGRect(x: 20, y: 26, width: 14, height: 14)
+        button.addTarget(self, action: #selector(cancellationView), for: .touchUpInside)
+        button.tintColor = .black
+        return button
+    }()
+
+    private let wreathImage: UIImageView = {
+        let image = UIImageView()
+        image.image = UIImage(named: "венок")
+        image.frame = CGRect(x: 92, y: 58, width: 200, height: 86)
+        return image
+    }()
+
+    private let nameCafeLabel: UILabel = {
+        let label = UILabel()
+        label.text = Constant.nameCafeLabelText
+        label.textAlignment = .center
+        label.numberOfLines = 0
+        label.frame = CGRect(x: 75, y: 184, width: 235, height: 128)
+        label.font = UIFont(name: "AmaticSC-Bold", size: 50)
```

Comment:
```suggestion
        label.font = UIFont(name: "", size: 50)
```
в. этом файле тоже все в константы